offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/OfferOverviewView.groovy

```diff
@@ -128,8 +129,9 @@ class OfferOverviewView extends FormLayout {
     }
 
     private void setupGrid() {
-        Column<Offer, Date> dateColumn = overviewGrid.addColumn({ overview -> overview.getModificationDate() })
-                .setCaption("Creation Date").setId("CreationDate")
+        Column<Offer, Date> dateColumn = overviewGrid.addColumn({ overview -> overview
+                .getModificationDate() }).setCaption("Creation Date").setId("CreationDate")
+        dateColumn.setRenderer(date -> date,  new DateRenderer('%1$tY-%1$tm-%1$td'))
```

Comment:
We have the ISO date format already in the data-model-lib.

Reply:
I dont want the time though

Reply:
Ah, I see. I think here we should tackle the issue rather than the symptoms. The display is technically correct, the offer displayed just is not (type-wise)
If we want to always only consider the date, currently done in the database, then the created offer itself should only contain the date.
Otherwise, we differentiate here between two different types of offer objects where they should be the same. What do you think?

Reply:
Since there is not direct requirement, we only change the representation to the user.